components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/gen/java/org/wso2/carbon/apimgt/rest/api/admin/ApiException.java

```diff
@@ -1,6 +1,6 @@
 package org.wso2.carbon.apimgt.rest.api.admin;
 
-@javax.annotation.Generated(value = "org.wso2.maven.plugins.JavaMSF4JServerCodegen", date = "2017-03-22T12:24:12.664+05:30")
+@javax.annotation.Generated(value = "org.wso2.maven.plugins.JavaMSF4JServerCodegen", date = "2017-04-07T14:13:41.057+05:30")
```

Comment:
do u need to change this file? there is no changes other than the generated time right? think we should omit this file